.github/workflows/autoupdate-preview.yaml

```diff
@@ -56,7 +60,20 @@ jobs:
             ## Using early preview changes
 
             ```sh
-            go get go.mongodb.org/atlas-sdk@dev-latest   
+            go get github.com/mongodb/atlas-sdk-go@dev-latest   
```

Comment:
Kind reminder that we have internal wiki that might need update post merge.

Reply:
yes, I was planning to update wiki once this PR is merged